libcontainer/nsenter/cloned_binary.c

```diff
@@ -394,6 +400,99 @@ static int seal_execfd(int *fd, int fdtype)
 	return -1;
 }
 
+struct bindfd_child_args {
+	int sockfd;
+	const char *mount_target;
+};
+
+static int bindfd_in_subprocess(void *arg)
+{
+	/*
+	 * In the interests of efficiency (read: minimizing the syscall count)
+	 * and conciseness, no attempt is made to release resources which would
+	 * be cleaned up automatically on process exit, i.e. when this function
+	 * returns. This includes filesystem mounts, as this function is
+	 * executed in a dedicated mount namespace.
+	 */
```

Comment:
It would be nice to mention that this function is run in a vfork, so folks need to keep in mind they can't use stuff that is forbidden in vfork.

Reply:
@cyphar ^^^

Reply:
Fair enough.